tests/integration/records/expected_column_types.py

```diff
@@ -134,8 +140,8 @@
     ],
     ('bigquery', 'redshift'): [
         'BIGINT', 'VARCHAR(256)', 'VARCHAR(256)', 'VARCHAR(256)', 'VARCHAR(256)',
-        'VARCHAR(256)', 'VARCHAR(256)', 'DATE', 'VARCHAR(8)',
-        'TIMESTAMP WITHOUT TIME ZONE', 'TIMESTAMP WITH TIME ZONE'
+        'VARCHAR(256)', 'VARCHAR(256)', 'VARCHAR(256)', 'VARCHAR(8)', 'VARCHAR(256)',
+        'VARCHAR(256)'
```

Comment:
Downside: No date/time types.

Upside: Can handle larger tables, as we're not working through SELECT and Pandas!